### PR TITLE
Contact-form

### PR DESCRIPTION
**Pull request by Hirwacardinalloic**
## Description
In this PR, we create a Contact Form.

## General requirements
 - [x] No linter errors.
 - [x] Follow GitHub flow.
 - [x] Professional Documentation.

## Project requirements, We:
 - [x] Added a Contact form with HTML and  CSS

Happy reviewing 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -299,5 +299,31 @@ <h4 class="ico">LET’S CONNECT</h4>
...
+      <section class="contact-form">
+        <div class="contact-section">
+          <section class="contact-content">
+            <h1>Contact me</h1>
+           <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+       
+         <form action="https://formspree.io/f/mdojqrgq " method="post">
+           <ul class="form-sections">
+             <li>
+               <input  required max="30" type="text" name="user-name" placeholder="Name" class="form-input">
```
Comment (zieeco) [inline index.html]: - Hey, let's kindly improve on our max to `maxlength="30"` as this is the correct way of adding the maxlength attribute to enforce the input's validation.

File: style.css [hunk 2]
```diff
@@ -325,3 +324,81 @@ body {
...
+  text-align: center;
+  line-height: 28px;
+  letter-spacing: 0.01rem;
+}
+
+.form-sections {
+  margin-top: 2rem;
+  list-style-type: none;
+  padding: 24px;
+}
```
Comment (zieeco) [inline style.css]: - Hey, there's an additional padding-left: 40px been applied to your form which is making your design look slightly different compared to the Figma design. kindly check to remove it to improve your design. see the image below to gain clarity.






Screenshot from 2022-12-15 01-27-35

File: style.css [hunk 3]
```diff
@@ -325,3 +324,81 @@ body {
...
   color: #acb7c3;
 }
+
+/* contact form */
+.contact-section {
+  background-color: #6070ff;
+  background-image: url(/images/contact-graph-2.png), url(/images/contact-graph-1.png), url(/images/contact-circle-1.png);
+  background-repeat: no-repeat, no-repeat, no-repeat;
+  background-size: 95px, 100px, 28px;
+  background-position: 77% 95%, 110% 3%, 40% 10%;
```
Comment (zieeco) [inline style.css:334]: - The form background image is not showing up as intended, please, kindly double-check to ensure you've fixed this issue.

Comment (zieeco) [reply style.css:334]: ### README FILE


- Please, kindly update your readme to follow the current readme [template](https://github.com/microverseinc/readme-template) structure and format to improve and make your readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply style.css]: Fixed.

File: style.css (hunk 3 above)
Comment (Abenezer-Tilahun) [reply style.css:334]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Hirwacardinalloic 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ The Form Submitted Successfully 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**